banderwagon/element_test.go

```diff
@@ -301,19 +314,15 @@ func TestBatchNormalize(t *testing.T) {
 		}
 
 		var expectedA, expectedB Element
-		expectedA.Set(&A).Normalise()
-		expectedB.Set(&B).Normalise()
-
-		BatchNormalize([]*Element{&A, &B})
-
-		if !A.Equal(&expectedA) {
-			t.Fatal("expected point `A` is incorrect ")
+		if err := expectedA.Set(&A).Normalise(); err != nil {
+			t.Fatalf("could not normalize point A: %s", err)
+		}
+		if err := expectedB.Set(&B).Normalise(); err == nil {
+			t.Fatal("points at infinity can't be normalized")
 		}
```

Comment:
Note that now if expect `Normalise` to fail, since this sub-tests is testing the behavior in points at infinity.